logcheck/testdata/src/onlyAllowContextual/onlyAllowContextual.go

```diff
@@ -29,4 +29,5 @@ func doNotAlllowKlog() {
 	klog.InfoS("test log")       // want `function "InfoS" should not be used, convert to contextual logging`
 	klog.ErrorS(nil, "test log") // want `function "ErrorS" should not be used, convert to contextual logging`
 	klog.V(1).Infof("test log")  // want `function "V" should not be used, convert to contextual logging` `function "Infof" should not be used, convert to contextual logging`
+	klog.KObjs(nil)              // want `function "KObjs" should not be used, convert to contextual logging`
```

Comment:
I think we want a better warning here. "convert to contextual logging" is misleading when printed for a line which uses contextual logging.

This implies that we need separate code for it and cannot simply add it to the blacklisted functions.

Reply:
At that point it probably needs a separate flag for enabling the check.

I'm leaning towards enabling it by default, even if that means that we need to update code in Kubernetes before we can switch to the newer logtools, or disable the check initially in Kubernetes' logcheck.conf.

Reply:
I am fine with it. Do you have any preferences on how you would like this to look like @pohly  ?

Reply:
No, not really.